js/popup.js

```diff
@@ -29,23 +29,25 @@ $(document).ready(function() {
       current_tab_id = t.id;
       //verfies that the current tab selected is the tab last queried
       if(wishlist_request.tab_id == current_tab_id) {
-        $("span.price").text(Number(wishlist_request.total).toFixed(2));
+        $(".wishlist_total").text(Number(wishlist_request.total).toFixed(2));
         wishlist_id = wishlist_request.wishlist_id;
       } else {
         var wishlist_data = getWishlistData(current_tab_id);
         if (!(wishlist_data.total).isNaN) {
-          $("span.price").text(Number(wishlist_data.total).toFixed(2));
-          $("span.wishlist_name").text(wishlist_data.name);
-          $("span.item_number").text(wishlist_data.item_num);
+	  $(".wishlist_loading").hide();
```

Comment:
The loading text is currently only showing like... the first time I guess. This is probably the wrong condition to stick this under. I think what's happening is that we hide the loading message and show the total info for the last time we calculated it, rather than indicating that we're recalculating.